_posts/dusan_andric/2018-05-25-icgc-jupyter.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Jupyter in ICGC"
+breadcrumb: true
+author: dusan_andric
+date: 2018-05-25
+categories: dusan_andric
+tags:
+    - ICGC
+    - Collaboratory
+    - Jupyter
+    - Analysis
+    - Data Science
+    - Overture
+teaser:
+    info: The ICGC Data Portal gets a new analysis tool! 
+    image: dusan_andric/jupyter/jupyter.png
+header: 
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Introduction
+
+After a successful soft launch, the ICGC DCC Software Engineering is proud to announce the availability of Jupyter notebooks as a new tool in the analysis toolbox within the ICGC Data Portal. 
+
+Users with DACO approval to the ICGC dataset will have the ability to login using their Google credentials. Once inside, users are presented with a familiar looking Jupyter Notebook environment with some value adds provided by us. 
+
+Our goal is to provide researchers and data scientists a way to programmatically explore the available ICGC data and get their hands dirty with some data science and analysis in a way that is sharable and reproducable. 
+
```

Comment:
The three first paragraphs of the intro, although perfectly fine individually, sounds to me a tiny bit too disconnected in terms of overall flow. Maybe just invert paragraphs 2 <-> 3 would be sufficient though.